### PR TITLE
enable blur for qt 6 and set qt 6 as default

### PR DESCRIPTION
**Pull request by sanjay-kr-commit**
Plasma 6 is will be released in few days so i tried porting blur effect

I don't usually code in cpp 
so my changes may seem messy 

sorry in advance for that

### CODE REVIEW

File: Kvantum/CMakeLists.txt
Comment (AlekEagle) [inline Kvantum/CMakeLists.txt:1]: Also include the building of the manager, preview, and themes with qt6 enabled

Comment (sanjay-kr-commit) [reply Kvantum/CMakeLists.txt:1]: enabled building of manager , preview and themes against qt6 by default

**Comment by tsujan** [comment]
Please see https://github.com/tsujan/Kvantum/issues/943#issuecomment-1971141666

**Comment by tsujan** [comment]
> I'm sorry I didn't know about the above message

No need to be sorry. I just want to prevent duplications.

I'll add a complete commit (most of which is ready) after I get KF6 and test it for a day or two. Then, if everything is fine, I'll release 1.1.0 with it.

**Comment by tsujan** [comment]
KF6 came here now. Added the commit: https://github.com/tsujan/Kvantum/commit/fc4ffbfe32f1519734519c4de0b4358b2aa5e8bd

Please test the latest git with Qt6 (and if you have Qt5 apps, with Qt5 too). Although I'm using it, I don't have Plasma6/Kwin6 yet.

**Pull request closed without merging**